src/tmx/Asn_J2735/include/asn_j2735_r63/TestMessage01.h

```diff
@@ -21,13 +21,13 @@ extern "C" {
 #endif
 
 /* Forward declarations */
-struct Header;
-struct Reg_BasicSafetyMessage;
+// struct Header;
```

Comment:
If these are no longer being used lets remove them

Reply:
I would leave it for now in case future changes.